src/main/java/com/yahoo/bullet/record/TypedBulletRecord.java

```diff
@@ -8,12 +8,14 @@
 import com.yahoo.bullet.typesystem.Type;
 import com.yahoo.bullet.typesystem.TypedObject;
 
+import java.io.Serializable;
+
 public abstract class TypedBulletRecord extends BulletRecord<TypedObject> {
     private static final long serialVersionUID = 5093501631768714558L;
 
     @Override
     protected TypedObject convert(Object object) {
```

Comment:
Nope, that breaks all the interfaces with the set*Map and set*List since Map and List are not serializable. I don't want to enforce a HashMap or ArrayList or some particular Serializable thing. I also don't want to copy them into one. I figure I can just let subclasses implement that if they want to using this